pygac_fdr/metadata.py

```diff
@@ -203,9 +203,11 @@ def _get_equator_crossings(self, ds):
         Returns:
             Longitudes and UTC times of first and second equator crossing, if any, NaN else.
         """
+        lat = ds['latitude'].load()  # load dataset to prevent netCDF4 indexing error
+
         # Use coordinates in the middle of the swath
-        mid_swath = ds['latitude'].shape[1] // 2
-        lat = ds['latitude'].isel(x=mid_swath)
+        mid_swath = lat.shape[1] // 2
+        lat = lat.isel(x=mid_swath)
```

Comment:
Is this correcting a bug in master?

Reply:
Yes, this occurs with the unstable dependencies only

Reply:
I didn't look into it any further, just did what the error message suggested...

> The indexing operation you are attempting to perform is not valid on netCDF4.Variable object. Try loading your data into memory first by calling .load().

Reply:
Okay, I will check my log-files to see if I had this issue.

Reply:
My setup is not affected... phew!